src/System.Security.Cryptography.Encoding/tests/Base64TransformsTests.cs

```diff
@@ -253,5 +253,45 @@ public static void ValidateWhitespace(string expected, string data)
                 Assert.Throws<FormatException>(() => cs.Read(outputBytes, 0, outputBytes.Length));
             }
         }
+
+        [Fact]
+        public void Blocksizes_ToBase64Transform()
+        {
+            using (var transform = new ToBase64Transform())
+            {
+                Assert.Equal(3, transform.InputBlockSize);
+                Assert.Equal(4, transform.OutputBlockSize);
+            }
+        }
+
+        [Fact]
+        public void Blocksizes_FromBase64Transform()
+        {
+            using (var transform = new FromBase64Transform())
+            {
+                Assert.Equal(4, transform.InputBlockSize);
+                Assert.Equal(3, transform.OutputBlockSize);
+            }
+        }
+
+        [Fact]
+        public void TransformUsageFlags_ToBase64Transform()
+        {
+            using (var transform = new ToBase64Transform())
+            {
+                Assert.False(transform.CanTransformMultipleBlocks);
```

Comment:
Is ToBase64Transform already effectively implemented as "true" here? (TransformBlock accepting an integer multiple of the input block size, TransformFinalBlock accepting any arbitrary amount of data) Or would flipping it to true actually cause failures?

Reply:
> Is ToBase64Transform already effectively implemented as "true" here?

No it's not. It just transforms one block at a time.
https://github.com/dotnet/corefx/blob/886b05b77245f5f738e10a64eb118183855b8365/src/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/Base64Transforms.cs#L37-L39
https://github.com/dotnet/corefx/blob/886b05b77245f5f738e10a64eb118183855b8365/src/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/Base64Transforms.cs#L65-L71

See also https://github.com/dotnet/corefx/issues/39583#issuecomment-513304310 for this.

> Or would flipping it to true actually cause failures?

This test https://github.com/dotnet/corefx/blob/886b05b77245f5f738e10a64eb118183855b8365/src/System.Security.Cryptography.Encoding/tests/Base64TransformsTests.cs#L101 will fail with these https://github.com/dotnet/corefx/blob/886b05b77245f5f738e10a64eb118183855b8365/src/System.Security.Cryptography.Encoding/tests/Base64TransformsTests.cs#L22 test-data.